AnomalyDetection/LibLinear/src/test/java/org/tribuo/anomaly/liblinear/LibLinearAnomalyTrainerTest.java

```diff
@@ -69,4 +79,33 @@ public void gaussianDataTest() {
         Helpers.testModelProtoSerialization(model,Event.class,testData);
     }
 
+    @Test
+    public void loadProtobufModel() throws IOException, URISyntaxException {
+        Path path = Paths.get(LibLinearAnomalyTrainerTest.class.getResource("liblinear-anomaly-431.tribuo").toURI());
+        try (InputStream fis = Files.newInputStream(path)) {
+            ModelProto proto = ModelProto.parseFrom(fis);
+            LibLinearAnomalyModel model = (LibLinearAnomalyModel) Model.deserialize(proto);
+
+            assertEquals("4.3.1", model.getProvenance().getTribuoVersion());
+
```

Comment:
It seems we could be testing the inner model here with e.g.:

assertEquals(-1.0, model.getInnerModels().get(0).getBias());

I'm not sure if that is too in the weeds or not or maybe there's a good reason not to do this.

Reply:
You mean we could explicitly check for model properties we know to be true in 4.3.1? I was considering comparing them against the freshly trained versions that exist elsewhere in the system, but for things like SGD there's some difference between ARM & x86_64 which would mean the tests need the tolerance parameters fudging to pass.